tests/trace/test_objects_query_builder.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from weave.trace_server import trace_server_interface as tsi
+from weave.trace_server.objects_query_builder import (
+    ObjectQueryBuilder,
+    _make_conditions_part,
+    _make_limit_part,
+    _make_object_id_conditions_part,
+    _make_offset_part,
+    _make_sort_part,
+)
+
+
+def test_make_limit_part():
+    assert _make_limit_part(None) == ""
+    assert _make_limit_part(10) == "LIMIT 10"
+    assert _make_limit_part(0) == "LIMIT 0"
+
+
+def test_make_offset_part():
+    assert _make_offset_part(None) == ""
+    assert _make_offset_part(5) == "OFFSET 5"
+    assert _make_offset_part(0) == "OFFSET 0"
+
+
+def test_make_sort_part():
+    assert _make_sort_part(None) == ""
+    assert _make_sort_part([]) == ""
+
+    sort_by = [tsi.SortBy(field="created_at", direction="asc")]
+    assert _make_sort_part(sort_by) == "ORDER BY created_at ASC"
+
+    sort_by = [
+        tsi.SortBy(field="created_at", direction="desc"),
+        tsi.SortBy(field="object_id", direction="asc"),
+    ]
+    assert _make_sort_part(sort_by) == "ORDER BY created_at DESC, object_id ASC"
+
+    # Invalid sort fields should be ignored
+    sort_by = [tsi.SortBy(field="invalid_field", direction="asc")]
```

Comment:
not error?

Reply:
hmmm, yeah lets error

Reply:
This is really just testing the internal method, the way to add a sort by field is with `builder.add_order` and the impl checks correctness and throws there